build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.CS.targets

```diff
@@ -1,7 +1,23 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003"> 
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <Target Name="GenerateBootstrapCS" 
-            BeforeTargets="BeforeCompile">
+    <PropertyGroup Condition="'$(WindowsAppSDKBootstrapAutoInitialize)'==''">
```

Comment:
why are we introducing another opt-out property?  we already have WindowsAppSdkBootstrapInitialize for conditionally importing the C#/C++ autoinit logic

Reply:
Ah. Didn't know we already had that. Distasteful (implies all bootstrap, not just autoinitialization) but OK, will use that.